deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/QAProvisioning.py

```diff
@@ -91,6 +116,9 @@ def process_deployment_data(self):
                                                             install_mode=install_type, hosts=current_host)
 
                 deployment_instance.install()
+                sleep(60)
```

Comment:
Put a comment above, indicating the reason for that sleep

Reply:
Done in 
https://github.com/wazuh/wazuh-qa/pull/1715/commits/d2d6f33566218b13fa077808ceece2ea1ff71a50